cmd/tfsec/main.go

```diff
@@ -8,7 +8,21 @@ import (
 	"github.com/aquasecurity/tfsec/internal/app/tfsec/cmd"
 )
 
+const transitionMsg = `
+======================================================
+tfsec is joining the Trivy family
+
+tfsec will continue to remain available 
+for the time being, although our engineering 
+attention will be directed at Trivy going forward.
+
+You can read more here: 
+https://github.com/aquasecurity/tfsec/discussions/1994
+======================================================
+`
+
 func main() {
+	fmt.Print(transitionMsg)
```

Comment:
@simar7 @itaysk Can we print it out as standard error? The standard out message disable us to parse the result of JSON format.

Reply:
I opened a pull request. Let's discuss that there. 

https://github.com/aquasecurity/tfsec/pull/2099

Reply:
you're right, sorry. It shouldn't be printed in machine-readable formats like json, csv, junit, sarif, etc.